services/cacheService.js

```diff
@@ -0,0 +1,10 @@
+const cache = require('memory-cache')
+
+const set = (key, value, time = 7200000) => {
```

Comment:
We should set the `ttl` of different instances of data to be stored in cache as the config.
e.g.
```
cache: {
  ttl: {
      defaut: <defaultValue>,
      userCache: <userCacheTTL>
  }
}
```